lib/_stream_readable.js

```diff
@@ -18,7 +18,7 @@ var Buffer = require('buffer').Buffer;
 
 Readable.ReadableState = ReadableState;
 
-var EE = require('events').EventEmitter;
+var EE = require('events');
```

Comment:
won't this break on older versions of Node since we're relying on Node's provided module? Or is this comment moot since this is the result of the regex pull from upstream?